src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -41,6 +41,15 @@
   <PropertyGroup>
     <DebugType Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">Full</DebugType>
   </PropertyGroup>
+ 
+  <!--
+     Generate config file for test project targeting .NET Framework. This config file has have binding redirect which
+	 is needed at time of running tests.
+   -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
```

Comment:
+1. 
https://github.com/search?l=&p=1&q=GenerateBindingRedirectsOutputType+user%3Adotnet+user%3Amicrosoft&ref=advsearch&type=Code&utf8=%E2%9C%93 is all I could find.